Bottom: 267c2e2cc5746f78de1552aa408597ace9c9f762
Top:    b0584ce716ef0997419f04c59d9a6f5617501b4c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 15:39:34 +0100

hid/gtk: Remove unused function ghid_button_set_text()


---

diff --git a/src/hid/gtk/gui-utils.c b/src/hid/gtk/gui-utils.c
index 8a75cae..a4d7b12 100644
--- a/src/hid/gtk/gui-utils.c
+++ b/src/hid/gtk/gui-utils.c
@@ -191,15 +191,6 @@ ghid_map_color_string (char *color_string, GdkColor * color)
 }
 
 
-void
-ghid_button_set_text (GtkWidget * button, gchar * text)
-{
-  GtkWidget *label = gtk_bin_get_child (GTK_BIN (button));
-
-  gtk_label_set_text (GTK_LABEL (label), text);
-}
-
-
 gchar *
 ghid_entry_get_text (GtkWidget * entry)
 {
diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 73e970e..dfed2c6 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -357,7 +357,6 @@ gboolean ghid_shift_is_pressed (void);
 void ghid_draw_area_update (GHidPort * out, GdkRectangle * rect);
 gchar *ghid_get_color_name (GdkColor * color);
 void ghid_map_color_string (gchar * color_string, GdkColor * color);
-void ghid_button_set_text (GtkWidget * button, gchar * text);
 gchar *ghid_entry_get_text (GtkWidget * entry);
 void ghid_check_button_connected (GtkWidget * box, GtkWidget ** button,
 				  gboolean active, gboolean pack_start,
